packages/frontend/core/src/components/page-list/list.tsx

```diff
@@ -61,7 +61,9 @@ const useItemSelectionStateEffect = () => {
         if (
           target.tagName === 'BUTTON' ||
           target.tagName === 'INPUT' ||
-          (e.target as HTMLElement).closest('button, input, [role="toolbar"]')
+          (e.target as HTMLElement).closest(
+            'button, input, [role="toolbar"], [data-all-doc-list="true"]'
```

Comment:
I think maybe you can reuse `[data-testid=page-list-item]`

Reply:
I remember that data-test-id seemed to be removed in the production environment.

Reply:
I think I remembered it wrongly.

Reply:
Good point. Or perhaps adding button role to the item list and then filter the event based on roles